GCP_deploy/template/index.html

```diff
@@ -0,0 +1,167 @@
+<!DOCTYPE html>
+<html >
+<!--From https://codepen.io/frytyler/pen/EGdtg-->
+<head>
+  <meta charset="UTF-8">
+  <title>Sentence Summarizer</title>
+  <link href='https://fonts.googleapis.com/css?family=Pacifico' rel='stylesheet' type='text/css'>
+<link href='https://fonts.googleapis.com/css?family=Arimo' rel='stylesheet' type='text/css'>
+<link href='https://fonts.googleapis.com/css?family=Hind:300' rel='stylesheet' type='text/css'>
+<link href='https://fonts.googleapis.com/css?family=Open+Sans+Condensed:300' rel='stylesheet' type='text/css'>
+  
+<meta name="viewport" content="width=device-width, initial-scale=1">
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+
+<body>
+
+ <div style="justify-content: center; text-align: center; margin-top: 5rem;" class="login">
+    <form action="{{ url_for('predict')}}" method="post" id="form1">
+        
+        <div class="wrap">
+            <h1>Sentence Summarizer</h1>
+   <div class="search">
```

Comment:
reformat to make sure tags are aligned

<div ..>
  <h1> ...
  <div ...>
  </div>
</div>